packages/ra-ui-materialui/src/button/CreateButton.tsx

```diff
@@ -62,6 +62,7 @@ const CreateButton: FC<CreateButtonProps> = props => {
         <Button
             component={Link}
             to={location}
+            classes={classesOverride}
```

Comment:
You should pass `classes` here as it actually merge the `classesOverride` and the default classes. This is why we pass the props (and not `rest`) to the `useStyles` hook

Reply:
You right! Sorry I will update it.